examples/cron-backfill.yaml

```diff
@@ -0,0 +1,114 @@
+# This is an example of back-filling day for a daily cron workflow.
```

Comment:
A link to `docs/cron-backfill.md` would be useful here for users who find this example independently from the docs.

Reply:
done